pkg/operator/periodicbackupcontroller/periodicbackupcontroller.go

```diff
@@ -272,3 +354,152 @@ func newCronJob() (*batchv1.CronJob, error) {
 
 	return obj.(*batchv1.CronJob), nil
 }
+
+func createBackupServerDaemonSet(cr backupv1alpha1.Backup, endpoints []string) *appv1.DaemonSet {
+	ds := appv1.DaemonSet{
+		ObjectMeta: v1.ObjectMeta{
+			Name:      backupServerDaemonSet,
+			Namespace: operatorclient.TargetNamespace,
+			Labels: map[string]string{
+				backupDSLabelKey: backupDSLabelValue,
+			},
+		},
+		Spec: appv1.DaemonSetSpec{
+			Selector: &v1.LabelSelector{
+				MatchLabels: map[string]string{
+					backupDSLabelKey: backupDSLabelValue,
+				},
+			},
+			Template: corev1.PodTemplateSpec{
+				ObjectMeta: v1.ObjectMeta{
+					Labels: map[string]string{
+						backupDSLabelKey: backupDSLabelValue,
+					},
+				},
+				Spec: corev1.PodSpec{
+					Tolerations: []corev1.Toleration{
+						{
+							Key:    masterNodeSelector,
+							Effect: corev1.TaintEffectNoSchedule,
+						},
+					},
+					NodeSelector: map[string]string{masterNodeSelector: ""},
+					Volumes: []corev1.Volume{
+						{Name: etcdDataDirVolName, VolumeSource: corev1.VolumeSource{
+							HostPath: &corev1.HostPathVolumeSource{
+								Path: etcdDataDirVolPath,
+								Type: ptr.To(corev1.HostPathUnset)}}},
+
+						{Name: etcdConfigDirVolName, VolumeSource: corev1.VolumeSource{
+							HostPath: &corev1.HostPathVolumeSource{
+								Path: etcdConfigDirVolPath}}},
+
+						{Name: etcdAutoBackupDirVolName, VolumeSource: corev1.VolumeSource{
+							HostPath: &corev1.HostPathVolumeSource{
+								Path: etcdAutoBackupDirVolPath}}},
+
+						{Name: etcdCertDirVolName, VolumeSource: corev1.VolumeSource{
+							HostPath: &corev1.HostPathVolumeSource{
+								Path: "/etc/kubernetes/static-pod-resources/etcd-certs"}}},
+					},
+					Containers: []corev1.Container{
+						{
+							Name:    etcdBackupServerContainerName,
+							Image:   os.Getenv("OPERATOR_IMAGE"),
+							Command: []string{"cluster-etcd-operator", "backup-server"},
+							Env: []corev1.EnvVar{
+								{
+									Name: nodeNameEnvVar,
+									ValueFrom: &corev1.EnvVarSource{
+										FieldRef: &corev1.ObjectFieldSelector{
+											FieldPath: nodeNameFieldPath,
+										},
+									},
+								},
+							},
+							Args: constructBackupServerArgs(cr, strings.Join(endpoints, ",")),
+							VolumeMounts: []corev1.VolumeMount{
+								{Name: etcdDataDirVolName, MountPath: etcdDataDirVolPath},
+								{Name: etcdConfigDirVolName, MountPath: etcdConfigDirVolPath},
+								{Name: etcdAutoBackupDirVolName, MountPath: etcdAutoBackupDirVolPath},
+								{Name: etcdCertDirVolName, MountPath: etcdCertDirVolPath},
+							},
+							TerminationMessagePolicy: corev1.TerminationMessageFallbackToLogsOnError,
+							ImagePullPolicy:          corev1.PullIfNotPresent,
+							SecurityContext: &corev1.SecurityContext{
+								Privileged: ptr.To(true),
```

Comment:
- Maybe no need to set privileged=true option, please see https://github.com/openshift/cluster-etcd-operator/pull/1183
- Give the container the corresponding authority other than privileged

Reply:
fixed 👍🏽

Reply:
@lance5890 so removing the policy causes issues to access `kube` manifests .. the security is must here

Reply:
u mean the /etc/kubernetes/manifests dir ? could u show the me the log of policy issues

Reply:
so the issue was `permission denied` to access the folder and its contents